pom.xml

```diff
@@ -26,12 +25,25 @@
     <properties>
         <revision>3.2.13</revision>
         <changelist>999999-SNAPSHOT</changelist>
-        <jenkins.version>1.609.1</jenkins.version>
+       <jenkins.version>2.332.4</jenkins.version> <!-- used updated version of jenkins -->
```

Comment:
Please fix the indentation here as well.

Reply:
fixed